extension-push/src/java/com/defold/push/Push.java

```diff
@@ -342,10 +342,6 @@ private void registerFirebase(Activity activity) {
             Log.w(Push.TAG, "Gcm Sender Id must be set.");
             return;
         }
-        FirebaseOptions.Builder builder = new FirebaseOptions.Builder()
```

Comment:
This is not recommended. I'm pretty sure it would cause problems if used in combination with extension-firebase-analytics for instance.

The documentation for FirebaseApp: https://firebase.google.com/docs/reference/android/com/google/firebase/FirebaseApp

"For a vast majority of apps, FirebaseInitProvider will handle the initialization of Firebase for the default project that it's configured to work with, via the data contained in the app's google-services.json file. This ContentProvider is merged into the app's manifest by default when building with Gradle, and it runs automatically at app launch. No additional lines of code are needed in this case."